CHANGELOG.md

```diff
@@ -1,3 +1,42 @@
+#### 0.5.9 (2023-05-03)
+
+##### Build System / Dependencies
```

Comment:
```suggestion
# 0.5.9 (2023-05-03)

## Build System / Dependencies
```

The version number is heading 1 and the other headings are heading 2. Please make the change for the rest of the headings added in this PR.